securityscanutils/issuewriter/local_writer.go

```diff
@@ -0,0 +1,47 @@
+package issuewriter
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v32/github"
+)
+
+type LocalIssueWriter struct {
+	repo      GithubRepo
+	outputDir string
+}
+
+var liw IssueWriter = &LocalIssueWriter{}
```

Comment:
Given that this is a new implementation of an interface, I think it would be helpful to have a simple suite of unit tests that define the contract of this struct. Tests like:
- Can write to file for validate release
- Writes to separate files for subsequent releases
- Appends contents to file, if one already exists (is this the desired behavior)?